electron/app/utils/colors.ts

```diff
@@ -0,0 +1,48 @@
+import _ from "lodash";
```

Comment:
This is great, best of both worlds.

Reply:
Was this intended to be an approval or just a comment?

Reply:
I was going to defer to @brimoor (specifically, the colors chosen) for approval. But I approve.

Reply:
The colors should be easy to change if we get a better list in the future